README.md

```diff
@@ -192,73 +161,14 @@ fn say_hello() {
 ```
 ~~~
 
-### File transclusion
-
-Yarner supports file transclusion to allow for more structured or modular project sources.
-*This feature is currently only supported for Markdown.*
-
-Here, the content of file `src/main.rs.md` would be pulled into the document
-before compilation:
-
-```md
-@{{[src/main.rs](src/main.rs.md)}}
-```
-which would render as a link in the sources file. Or simply:
-```md
-@{{src/main.rs.md}}
-```
-which would not render as a link.
-
-Unnamed entrypoints are renamed to file entrypoints during transclusion.
-It is not required that transcluded files have their own entrypoints.
-
-All code blocks from transcluded files are accessible from the *transcluding* file,
-as well as from other transcluded files.
-
-### Include linked files
-
-Files linked from the main source document are included in the compilation process.
-*This feature is currently only supported for Markdown.*
-
-As an example, yarner would also compile the file `src/main.rs.md` here:
-
-```md
-* [src/main.rs](src/main.rs.md)
-```
-
-However, files are only included in the compilation if they are referenced by a *relative path*, and the file exists in that location.
-
-### Multiple languages
-
-Some documentation formats allow you to indicate the language that a code block is written in. In
-fact, it is recommended that you always include the language when you write a code block,
-particularly if multiple programming languages are used within the same document.
-
-By properly labelling all code blocks, it is then possible to write a program in multiple
-programming languages at once. Whether this is practical or not remains to be seen, but it is
-supported nonetheless. By then supplying a language name on the command line, only code blocks in
-that language are used when generating the tangled source. For example, here is a trivial program
-written in two languages:
-
-~~~tex
-Here we have hello world in Ruby:
-
-```rb
-puts "Hello world"
-```
-
-And here it is again in Rust:
+[File transclusions](docs/Features.md#file-transclusions) and [Links](docs/Features.md##include-linked-files) are further features that allow for projects with multiple code, documentation and/or source files.
 
-```rust
-fn main() {
-  println!("Hello world");
-}
-```
-~~~
+## Configuration
 
-Compiling this with no language supplied with just ignore language information, so a single output
-will be generated containing both languages. However, supplying the `--language rb` flag to yarner
-will cause only the code blocks tagged with `rb` will be used to generate code.
+Each style supports some additional configuration, which is provided via a toml configuration file
+(default: Yarner.toml). A file with default configurations is generated through the `create` sub-command.
+See the comments in these files for details on individual settings.
```

Comment:
@adamreichold Argh... not so easy, as the main file is filled in at runtime. Maybe I need a separate file for the config docs.

Reply:
Or use string replace...

Reply:
Yes I was anticipating to use have a placeholder in the template like `<<<MAIN_FILE>>>` that can then be processed using `template.replace("<<<MAIN_FILE>>>", actual_main_file)` at runtime?

Reply:
It is admittedly a kluge, but it usually works out fine and with the templates you do know if they should every contain `<<<MAIN_FILE>>>` with a different meaning than the placeholder.

Reply:
Resolved...